api_docs/changelog.md

```diff
@@ -20,6 +20,15 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 263**
+
+* [`GET /events`](/api/get-events): Added `status_end_time` field
+  to `user_status` object.
+* [`POST /register`](/api/register-queue): Added `status_end_time`
+  field to `user_status` objects.
+* [`POST /users/me/status`](/api/update-status): Added support for new
+  `status_end_time` parameter.
```

Comment:
I think [we decided to name this parameter `scheduled_end_time`](https://chat.zulip.org/#narrow/stream/378-api-design/topic/status.20end.20time.20to.20clear.20user.20status/near/1730239)

Reply:
Updated!